playwright/tests/smoke/flowContainer.spec.ts

```diff
@@ -0,0 +1,148 @@
+
+test.describe('Flow container', async () => {
+  let page: Page;
+  let flowContainerPage: FlowContainerPage;
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext({ viewport: { width: 1200, height: 850 } });
+    page = await context.newPage();
+    flowContainerPage = new FlowContainerPage(page);
+    await page.goto('/flow-container');
+  });
+
+  test('Flow container: 1 - checking presence of all Flow Container sections', async () => {
+    await flowContainerPage.toggleEditMode();
+    expect(await page.locator(flowContainerPage.flowContainer1Item).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainer25Width).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainer33Width).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainer50And100).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainer50Width).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainer66Width).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainer75Width).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainer100Only).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainerContentful).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainerLibrary).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainerLimitedFlow).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainerNestedFlow).isVisible()).toBeTruthy();
+    expect(await page.locator(flowContainerPage.flowContainerDefault).isVisible()).toBeTruthy();
+  });
+
+  test('Flow container: 2 - checking adding and filling in a Square Image in the Default Flow Container', async () => {
+    await page.click(flowContainerPage.flowContainerDefault);
+    await page.click(flowContainerPage.addFlowContainerButton);
+    await page.click(flowContainerPage.imagesCheckbox);
+    await page.click(flowContainerPage.squareImageButton);
+    await page.click(flowContainerPage.squareImageButtonInsideContainer);
+    await page.click(flowContainerPage.selectImageButton);
+    await page.setInputFiles('input[type=file]', flowContainerPage.pathToImages + flowContainerPage.imageTwoName);
+    await page.fill('#image-alt', 'alt-test');
+    await Promise.all([
+      page.waitForResponse(response => response.url()
+        .includes('flowContainer') && response.status() === 200),
+      page.click(flowContainerPage.submitButton),
+    ]);
+    const defaultContainer = page.locator(flowContainerPage.flowContainerDefault);
+    const img = defaultContainer.locator('img');
+    expect.soft(await img.getAttribute('alt')).toEqual('alt-test');
+    expect.soft(await img.getAttribute('src')).toContain(flowContainerPage.imageTwoName);
```

Comment:
@pomi ,

imagePathRegex should be checked 
`const imagePathRegex = new RegExp("images\/pages" + pagePath + "[a-zA-Z0-9]+\/" + imageName, "");`
please see 
https://github.com/johnsonandjohnson/Bodiless-JS/pull/584/files#diff-abfd3f9753c21f575bdae486babc4d2f7fd5129967c231e337515b047ea149c2R29

https://github.com/johnsonandjohnson/Bodiless-JS/pull/584/files#diff-abfd3f9753c21f575bdae486babc4d2f7fd5129967c231e337515b047ea149c2R99

===
the following should be added on global level (commented about it in the Migration to Playwright PR)
https://github.com/johnsonandjohnson/Bodiless-JS/pull/584/files#diff-abfd3f9753c21f575bdae486babc4d2f7fd5129967c231e337515b047ea149c2R100